js/logos.json

```diff
@@ -279,8 +275,7 @@
 		"title": "Terra Kulture",
 		"filename": "terra_kulture",
 		"category": [
-			"Ecommerce",
-			"NGO"
+			"Media and Entertainment"
```

Comment:
Can you split this into two?

Reply:
To add a new commit for Terra Kulture?

Reply:
Nah. If we'll ever need Media to be a different category from Entertainment, then we should split the categories.